app/controllers/admin/articles_controller.rb

```diff
@@ -44,8 +44,9 @@ def update
       article.email_digest_eligible = article_params[:email_digest_eligible].to_s == "true"
       article.boosted_additional_articles = article_params[:boosted_additional_articles].to_s == "true"
       article.boosted_dev_digest_email = article_params[:boosted_dev_digest_email].to_s == "true"
-      article.user_id = article_params[:user_id].to_i
-      article.co_author_ids = article_params[:co_author_ids].split(",").map(&:strip)
+      article.user_id = article_params[:user_id].to_i if article_params[:user_id].present?
```

Comment:
I added this guard clause here because while pairing with @atsmith813, he noticed that without the guard clause, this line was overwriting the user and `user_id`. 🔎